README.md

```diff
@@ -45,15 +46,33 @@ The core code of the project lays inside [flutter/lib](flutter/lib) directory an
 This project is license under a [GPL-3.0](https://www.gnu.org/licenses/gpl-3.0.html) license.
 
 
-## Disclaimer & Credits
+## Credits
 
-* This game is a **fan-made**, **non-commercial** recreation of Djambi,
```

Comment:
You are overriding my latest changes in the README file! Most probably you are doing your changes on top of old commit. Please use the latest.